olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/PartitionsRequest.h

```diff
@@ -37,6 +38,29 @@ namespace read {
  */
 class DATASERVICE_READ_API PartitionsRequest final {
  public:
+  /**
+   * @brief Sets the list of partitions.
```

Comment:
Please extend and specify what happens when you set to boost::none and also that the limit is 100 per request.